src/module.ts

```diff
@@ -100,7 +114,9 @@ const defaults: ModuleOptions = {
     idLength: 64,
     storePrefix: 'sessions',
     cookieSameSite: 'lax',
-    storageOptions: {}
+    storageOptions: {
+      driver: 'memory'
+    }
```

Comment:
Currently build fails because nuxt module builder fails to export proper types. So the types are instead inherited from the `defaults`. I suggest we quickly fix it in order to merge this PR and figure out nuxt module builder in the other issue.

```suggestion
    storageOptions: {
      driver: 'memory',
      options: {}
    }
```